### PR TITLE
Add custom fields docs

### PR DESCRIPTION
**Pull request by seancdavis**
### Updated Content

- [Custom fields feature guide](https://deploy-preview-369--stackbit-docs.netlify.app/features/visual-editing/custom-fields)
- [New section](https://deploy-preview-369--stackbit-docs.netlify.app/features/visual-editing#custom-fields) in [visual editing feature guides](https://deploy-preview-369--stackbit-docs.netlify.app/features/visual-editing)
- References for [`controlFilePath`](https://deploy-preview-369--stackbit-docs.netlify.app/reference/config/content-modeling/fields#controlfilepath) and [`controlType`](https://deploy-preview-369--stackbit-docs.netlify.app/reference/config/content-modeling/fields#controltype)
- References for [shared `controlType` values](https://deploy-preview-369--stackbit-docs.netlify.app/reference/config/content-modeling/fields#control-types) available on all fields.

### Content Review Notes

- Reference material above is mostly usage information and can be skimmed for language comments.
- The feature guide is completely new. It'd be helpful for me to see some feedback on usage, grammar, voice, etc. to start to better understand and conform to the Netlify voice.
- While we want to get this out quickly, it would also be helpful for me to understand what types of technical questions you might ask or feel like are missing from this content if going through a more formal process.

### Cute Animal Photo

image

### CODE REVIEW

**Comment by youvalv** [comment]
Task linked: [CU-8685e3duq Add custom fields docs](https://app.clickup.com/t/8685e3duq)

File: content/pages/features/visual-editing/custom-fields.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
+---
+title: Custom Fields
+excerpt: Create custom controls for fields in the page and content editors.
```
Comment (jmarlena) [inline content/pages/features/visual-editing/custom-fields.md:3]: **Rhetorical musing:** Curious if there's a clear distinction between "in the page" and "content editors" or if this is the same thing.

File: content/pages/features/visual-editing/custom-fields.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+toc:
+  hide: false
+  heading: 'On this page'
+hideBreadcrumbs: false
+hideSidebar: false
+tags:
+  - Features
+---
+
+Custom fields provide the ability to extend the default set of field types and controls available in the page and content editors.
```
Comment (jmarlena) [inline content/pages/features/visual-editing/custom-fields.md:15]: Meta feedback: I wonder if there are specific use cases this was set up for that we can highlight here or share as potential inspiration.

**Review by jmarlena — APPROVED** [review]
🚢 Looks sound to ship! 

Left 2 musing comments on what I was thinking as I read but tried to stick to a quick soundness review here. 💫

**Pull request merged**